### PR TITLE
Modifying migration documents for 3.1.0

### PR DESCRIPTION
**Pull request by HiranyaKavishani**
## Purpose
Renaming and modifying existing migration documentation for 3.1.0

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md [hunk 1]
```diff
@@ -295,12 +295,12 @@ current API Manager 2.0.0 version and run the below scripts against **the databa
         Changing these configuration should only be done before the initial API-M Server startup. If changes are done after the initial startup, the registry resource created previously will not be available.
 
 !!! note
-    If you are using WSO2 Identity Server (WSO2 IS) as a Key Manager, follow the instructions in [Upgrading WSO2 IS as the Key Manager to 5.9.0](../upgrading-wso2-is-as-key-manager/upgrading-from-is-km-520-to-590.md).
+    If you are using WSO2 Identity Server (WSO2 IS) as a Key Manager, follow the instructions in [Upgrading WSO2 IS as the Key Manager to 5.10.0](../upgrading-wso2-is-as-key-manager/upgrading-from-is-km-520-to-590.md).
 
 -   [Step 1 - Migrate the API Manager configurations](#step-1-migrate-the-api-manager-configurations)
--   [Step 2 - Upgrade API Manager to 3.0.0](#step-2-upgrade-api-manager-to-300)
+-   [Step 2 - Upgrade API Manager to 3.1.0](#step-2-upgrade-api-manager-to-300)
```
Comment (vithu30) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md]: Shall we update the version in the links too?
``` suggestion
-   [Step 2 - Upgrade API Manager to 3.1.0](#step-2-upgrade-api-manager-to-310)
```

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md [hunk 2]
```diff
@@ -470,7 +470,7 @@ Follow the instructions below to move all the existing API Manager configuration
 3.  To start the migration process, run the respective migration script based on your environment.
 
     ??? note "Linux/Mac OS"
-        Run the [apim200_to_apim300_gateway_artifact_migrator.sh](../../assets/attachments/install-and-setup/apim200_to_apim300_gateway_artifact_migrator.sh) script, as shown below, to migrate from WSO2 API Manager 2.0.0 to 3.0.0. 
+        Run the [apim200_to_apim300_gateway_artifact_migrator.sh](../../assets/attachments/install-and-setup/apim200_to_apim300_gateway_artifact_migrator.sh) script, as shown below, to migrate from WSO2 API Manager 2.0.0 to 3.1.0.
```
Comment (vithu30) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md]: Shall we rename the script file name?

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md [hunk 3]
```diff
@@ -2055,12 +2055,12 @@ This concludes the upgrade process.
...
+    If you are using a migrated API and wants to consume it via an application which supports JWT authentication (default type in API-M 3.1.0), you need to republish the API. Without republishing the API, JWT authentication doesn't work as it looks for a local entry which will get populated while publishing.
 
     You can consume the migrated API via an OAuth2 application without an issue.
 
 !!! Important
-    In API Manager 3.0.0, login flow is changed with role base access control where user should have respective roles to access Publisher and Developer Portal. Since 3.0.0 UI depends completely on REST APIs, authentication to access different components solely depends on the scope-role mapping defined in registry file `tenant-conf.json`. 
+    In API Manager 3.1.0, login flow is changed with role base access control where user should have respective roles to access Publisher and Developer Portal. Since 3.1.0 UI depends completely on REST APIs, authentication to access different components solely depends on the scope-role mapping defined in registry file `tenant-conf.json`.
```
Comment (vithu30) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md]: Shall we point the doc of adding role-mapping via admin portal feature here? WDYT?

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md (hunk 2 above)
Comment (HiranyaKavishani) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md]: +1

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md (hunk 1 above)
Comment (HiranyaKavishani) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md]: Modified with fd6185537f170e635be231dd5f58721f09d8ae6d

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md (hunk 2 above)
Comment (HiranyaKavishani) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md]: Adding fd6185537f170e635be231dd5f58721f09d8ae6d

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md (hunk 3 above)
Comment (HiranyaKavishani) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md]: +1. There are some other scripts changes too. This PR basically is to modify the existing migration documentations to 3.1.0. We will add the link as you suggest here when modifying the content inside them.

Comment (vithu30) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md]: ack

**Pull request merged**